.vscode/settings.json

```diff
@@ -6,6 +6,14 @@
     "*.targets": "xml",
     "*.tasks": "xml"
   },
+  "eslint.format.enable": true,
+  "eslint.workingDirectories": [
```

Comment:
Interesting that we need this + the package.json in the root of the Razor folder

Reply:
It is weird, but apparently needed. Without this updated eslint errors aren't show in VSCode.